commit 5ef8c33f318cdb836ada3459c78bdd26ac769427
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Sep 18 10:07:54 2017 +0100

    Revert "config: Enable CONFIG_MEMCG, but leave it disabled (due to memory cost). Enable with cgroup_enable=memory."
    
    This reverts commit 7dbc189967bef8e7f431c982557d0e73eb13f161.

diff --git a/kernel/cgroup.c b/kernel/cgroup.c
index 2650df194b5d..2c785f957024 100644
--- a/kernel/cgroup.c
+++ b/kernel/cgroup.c
@@ -5637,7 +5637,7 @@ int __init cgroup_init_early(void)
 	return 0;
 }
 
-static u16 cgroup_disable_mask __initdata = 1<<0;
+static u16 cgroup_disable_mask __initdata;
 
 /**
  * cgroup_init - cgroup initialization
@@ -6178,28 +6178,6 @@ static int __init cgroup_no_v1(char *str)
 }
 __setup("cgroup_no_v1=", cgroup_no_v1);
 
-static int __init cgroup_enable(char *str)
-{
-	struct cgroup_subsys *ss;
-	char *token;
-	int i;
-
-	while ((token = strsep(&str, ",")) != NULL) {
-		if (!*token)
-			continue;
-
-		for_each_subsys(ss, i) {
-			if (strcmp(token, ss->name) &&
-			    strcmp(token, ss->legacy_name))
-				continue;
-
-			cgroup_disable_mask &= ~(1 << i);
-		}
-	}
-	return 1;
-}
-__setup("cgroup_enable=", cgroup_enable);
-
 /**
  * css_tryget_online_from_dir - get corresponding css from a cgroup dentry
  * @dentry: directory dentry of interest
